mug-brownie.md

```diff
@@ -2,6 +2,8 @@ I love this recipe, because it's so simple to make.
 
 Often times, I will skip the microwave stage, and just eat the brownie mix. There no egg in it, so it's not even a salmonella risk!
 
+I discovered this recipe by watching Youtube.
+
```

Comment:
Youtube. is a great source of information, great. What was the exact video you watched?